com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md

```diff
@@ -36,7 +36,7 @@ These limitations provided the motivation towards the development of the Grid Se
 
 An image can be thought of as a matrix of a predefined width (W) and a height (H) and each pixel can be thought of as simply an array of length 3 (in the case of RGB), `[Red, Green, Blue]` holding the different channel information of the color (channel) intensities at that pixel location. Thus an image is just a 3 dimensional matrix of size WxHx3. A Grid Observation can be thought of as a generalization of this setup where in place of a pixel there is a "cell" which is an array of length N representing different channel intensities at that cell position. From a Convolutional Neural Network point of view, the introduction of multiple channels in an "image" isn't a new concept. One such example is using an RGB-Depth image which is used in several robotics applications. The distinction of Grid Observations is what the data within the channels represents. Instead of limiting the channels to color intensities, the channels within a cell of a Grid Observation generalize to any data that can be represented by a single number (float or int).
 
-Before jumping into the details of the Grid Sensor, an important thing to note is the agent performance and qualitatively different behavior over raycasts. Unity MLAgent's comes with a suite of example environments. One in particular, the [Food Collector](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Learning-Environment-Examples.md#food-collector), has been the focus of the Grid Sensor development.
+Before jumping into the details of the Grid Sensor, an important thing to note is the agent performance and qualitatively different behavior over raycasts. Unity MLAgent's comes with a suite of example environments. One in particular, the [Food Collector](../../docs/Learning-Environment-Examples.md#food-collector), has been the focus of the Grid Sensor development.
```

Comment:
Not sure if this is the best approach, but linking to master here will eventually brake.

Reply:
If we ever generate package docs for this, the relative link won't work then either. how about using `release_10_docs`?

Reply:
There are some other links in the extensions package that do "../.." should I also link them to release_10_docs? How can I make sure we will update these at each release?

Reply:
> should I also link them to release_10_docs

Sure

> How can I make sure we will update these at each release?

There's a pre-commit hook that looks for these already. You made the changes in https://github.com/Unity-Technologies/ml-agents/pull/4643